sdk_entrepot_gpf/scripts/entities.py

```diff
@@ -145,7 +146,22 @@ def action_upload_close(upload: Upload, mode_cartes: bool) -> None:
 
     @staticmethod
     def action_upload_checks(upload: Upload) -> None:
+        checks = upload.api_list_checks()
         Config().om.info(f"Bilan des vérifications de la livraison {upload} :")
+        if(len(checks["passed"]) != 0):
+            Config().om.info(f"\t * {len(checks['passed'])} vérifications passées:")
+            [Config().om.info(f"\t\t {verification['check']['name']}") for verification in checks["passed"]]
+        if(len(checks["asked"]) != 0):
+            Config().om.warning(f" * {len(checks['asked'])} vérifications en cours ou en attente:")
+            [Config().om.warning(f"\t {verification['check']['name']}") for verification in checks["asked"] + checks["in_progress"]]
+        if(len(checks["failed"]) != 0):
+            Config().om.info(f"\t * {len(checks['failed'])} vérifications échouées:")
+            for verification in checks["failed"]:
+                Config().om.error(f"\t {verification['check']['name']} ")
+                check = CheckExecution(verification)
```

Comment:
Dans le cas où le datastore n'est pas null, il faut le propager :

```suggestion
                check = CheckExecution(verification, datastore=upload.datastore)
```